manifest.php

```diff
@@ -34,12 +34,12 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '4.0.0',
+    'version' => '4.0.1',
```

Comment:
you need to increase at least the feature version : `4.1.0` if a dependency is increased by a breaking version